apps/wallet/src/ui/app/staking/validators/SelectValidatorCard.tsx

```diff
@@ -96,6 +109,7 @@ export function SelectValidatorCard() {
                             className={cl('group relative w-full cursor-pointer', {
                                 'rounded-xl bg-shader-neutral-light-8':
                                     selectedValidator?.address === validator.address,
+                                'opacity-50': !isAddressCommitteeMember(validator.address),
```

Comment:
is this needed because looks like the `selectedValidator` comes from the `validatorList` which is just  `sortedAsc const sortedAsc = validatorsRandomOrder.map ...` and  validatorsRandomOrder contains just comittee members  
```
const validatorsRandomOrder = useMemo(
        () => [...(data?.committeeMembers || [])].sort(() => 0.5 - Math.random()),
        [data?.committeeMembers],
    );
```
so it seams selectedValidator will always be a comittee Member so this check `isAddressCommitteeMember(validator.address)` will always be true

Reply:
Fixed in [a7531ec](https://github.com/iotaledger/iota/pull/6257/commits/a7531eceec81ec8d624a17a4c5b39a177625181d)